dev/src/user.clj

```diff
@@ -7,35 +7,46 @@
             [clojure.java.io :as io]
             [fluree.server.system :as system]
             [fluree.db.util.log :as log]
+            [aero.core :as aero]
+            [meta-merge.core :refer [meta-merge]]
             [integrant.core :as ig]
             [integrant.repl :refer [clear go halt init reset reset-all]]))
 
-;; Three Server Configuration
-(def server-1 "/ip4/127.0.0.1/tcp/62071")
-(def server-2 "/ip4/127.0.0.1/tcp/62072")
-(def server-3 "/ip4/127.0.0.1/tcp/62073")
-(def servers-str (str/join "," [server-1 server-2 server-3]))
-
-(def server-1-overrides
-  {:http/jetty        {:port 58090}
-   :fluree/connection {:storage-path "data/srv1"}
-   :fluree/consensus  {:servers servers-str
-                       :this-server server-1}})
-
-(def server-2-overrides
-  {:http/server       {:port 58091}
-   :fluree/connection {:storage-path "data/srv2"}
-   :fluree/consensus  {:consensus-servers     servers-str
-                       :consensus-this-server server-2}})
-
-(def server-3-overrides
-  {:http/server       {:port 58092}
-   :fluree/connection {:storage-path "data/srv3"}
-   :fluree/consensus  {:consensus-servers     servers-str
-                       :consensus-this-server server-3}})
-
-(def query-server-1-overrides
-  {:http/server {:port 58095}
-   :fluree/connection {:method :remote
-                       :servers "http://localhost:58090"}
-   :fluree/consensus {:consensus-type :none}})
+(defn load-config
```

Comment:
Some of [this functionality will be added in the config branch](https://github.com/fluree/server/compare/feature/user-config?expand=1#diff-2723cd4d3bc8b24ace602f5fbb8ab4e3bd1a67c7142d06bed0c093c777583d47R61-R71) when it's ready. I'm working on validation now, but I hope to be able to open a pr today.

Reply:
Ok, happy to adjust. Just working on REPL setup with the recent changes, and making sure multi-server raft is still working well, which is the more complex REPL setup.

Reply:
I opened #78 to store config in JSON. It might be easier to base this work off of that branch. It's also easier to do merges of configs because we don't have to worry about dissocing irrelevant keys (if you want to replace standalone with raft, for example). though that code isn't there yet, it will be easy to add when we need it.